src/src/components/ingest/publications_edit.jsx

```diff
@@ -1937,9 +1937,9 @@ class PublicationEdit extends Component {
     }
     if (
       this.props.editingPublication &&
-      this.props.editingPublication.data_types
+      this.props.editingPublication.dataset_type
     ) {
-      len = this.props.editingPublication.data_types.length;
+      len = this.props.editingPublication.dataset_type.length;
```

Comment:
`dataset_type` is a string

Reply:
the "renderAssayArray" function this is within is never used, holdover from dataset form